pyramid_openapi3/__init__.py

```diff
@@ -274,16 +287,21 @@ def register() -> None:
         config.add_route(route_name, f"{route}/{path.name}")
 
         custom_formatters = config.registry.settings.get("pyramid_openapi3_formatters")
+        custom_deserializers = config.registry.settings.get("pyramid_openapi3_deserializers")
 
         config.registry.settings[apiname] = {
             "filepath": filepath,
             "spec_route_name": route_name,
             "spec": spec,
             "request_validator": RequestValidator(
-                spec, custom_formatters=custom_formatters
+                spec,
+                custom_formatters=custom_formatters,
+                custom_media_type_deserializers=custom_deserializers
```

Comment:
```suggestion
                custom_media_type_deserializers=custom_deserializers,
```

Reply:
I think this is what trips `black`. Please run `black <path_to_file>` or `pipenv run black <path_to_file>` locally to be sure.